### ISSUE TITLE
[Feature] Thaumcraft Integration 

### ISSUE THREAD

**Issue by Kris1432** [TheCricket/Chisel-2#234]
It would be awesome if there were aspects in the blocks (& items)

**Comment by CptSpaceToaster** [comment]
This would have to be balanced quite carefully... as Thaumcraft's big deal is that each block and item can be "researched/investigated" once.  Chisel adds who knows how many blocks... making aspects incredibly easy to obtain if we were to add integration.

**Comment by Kris1432** [comment]
the alterations of vanilla blocks would just be already obtained when scanning one of them. for example: the cobblestone alterations would all be scanned if just one of them were to be scanned (or the cobblestone itself)

**Comment by CptSpaceToaster** [comment]
What of alterations that we link together, that have separate entries in Thaumcraft?

**Comment by TheCricket** [comment]
I have been looking into this and antiblocks already have terra. So I don't know exactly the reason all blocks don't work atm

**Comment by warlordjones** [comment]
I believe Azanor grabs aspects from recipes unless overriden - if there is something known to Thaumcraft in the recipe, it will include those aspects on the block/item. I'm speculating (from some experience) here.

**Comment by warlordjones** [comment]
Yeah, the aspects are grabbed via the recipe. I'm looking into a clean way to implement aspects for all blocks, that isn't just manually adding them for each block type, short example of which is in my latest commit. There's probably a way to do it, just can't think of it at the moment. Will discuss future on IRC later.
